horovod/common/operations.cc

```diff
@@ -1581,6 +1603,14 @@ void InitializeHorovodOnce(const int* ranks, int nranks) {
   while (!horovod_global.initialization_done) {
     std::this_thread::sleep_for(std::chrono::milliseconds(1));
   }
+
+#if HAVE_MLSL
+  LOG(DEBUG) << "BG-thread init done";
```

Comment:
Could this be useful information for non-MLSL scenarios as well?  Looks generic enough and could potentially provide insight into certain types of failures.  Thoughts, @alsrgv?

Reply:
Yeah, I'm fine to make remove `#if` around this log message.

Reply:
Done